Source/WebCore/rendering/RenderTableCol.cpp

```diff
@@ -38,6 +38,8 @@
 #include "RenderTable.h"
 #include "RenderTableCaption.h"
 #include "RenderTableCell.h"
+#include "rendering/RenderObject.h"
+#include "rendering/style/RenderStyle.h"
```

Comment:
This should just be:

`#include "RenderObject.h"` and `#include "RenderStyle.h"`